plutus-core/src/Language/PlutusCore/Evaluation/Machine/ExBudgeting.hs

```diff
@@ -12,6 +12,16 @@
 {-# LANGUAGE TypeFamilies           #-}
 {-# LANGUAGE UndecidableInstances   #-}
 
+{-# OPTIONS_GHC -XStrictData        #-}
```

Comment:
Why not `{-# LANGUAGE StrictData        #-}`?

Reply:
Pure ignorance!

Reply:
I've changed that but put it after all the other pragmas so that it didn't get buried in the noise.